docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst

```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
        $ aws s3api get-bucket-lifecycle-configuration --bucket zenko-bucket
 
 Once the lifecycle rules on the bucket are set, the rules apply to all
-objects in the specified bucket.
\ No newline at end of file
+objects in the specified bucket.
+
+.. _Lifecycle Queries:
+
+Querying Lifecycle Events
+-------------------------
+
+A lifecycle metrics feature is not available for release 1.1. You can,
+however, access the storage location of transitioned object data by
+viewing the object's metadata, for example by making a head-object
```

Comment:
~head-object request~ HEAD request

"head-object" is a reference to the command provided by the user via the command line, while the request itself would be referred by the HTTP request method (i.e. HEAD).

For example see: https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectHEAD.html

Reply:
done.